settings.xamlstyler

```diff
@@ -39,4 +39,4 @@
     "FormatOnSave": true,
     "CommentPadding": 2,
     "IndentSize": 4
-}
\ No newline at end of file
+}
```

Comment:
Undo the new line please.

Reply:
I was making this change through the github interface and it seems to be impossible to do through it.

Reply:
Yeah, it's a chore!!